.github/workflows/build.yml

```diff
@@ -0,0 +1,90 @@
+name: build
+
+env:
+  DOCKERHUB_ORG: mobilecoin
+  REPO_NAME: block-explorer
+  TARGET_CLUSTER: utility-r1-d-k8s
+  CHART_REPO: https://harbor.mobilecoin.com/chartrepo/mobilecoinofficial-private
+  CHART_PATH: ./.internal-ci/charts/block-explorer
+  REACT_APP_RESERVE_AUDITOR_URL: https://auditor.mobilecoin.foundation/api
```

Comment:
we can remove the env vars now that they are hardcoded into the app:
REACT_APP_RESERVE_AUDITOR_URL
REACT_APP_FULL_SERVICE_URL

Reply:
We probably don't want to hardcode paths?  Wouldn't we have different urls for a mainnet/testnet block explorer.  Ideally we have 1 container image we just configure to point at testnet or mainnet.

Reply:
ah, I didn't think about testnet. Right now the app is configured to try getting these values from the environment and fall back to the hardcoded values if it can't find them. When Ricky and Oz were getting this set up last week, I know they were having trouble getting these env vars to actually work in CI. The app will pick them up if there's a .env file present with them defined in it, but for some reason putting them in the dockerfile or some other places didn't work.

Reply:
If these are working, I think it's find to leave them in, but I don't think that's the case and they should be removed if they aren't working